### PR TITLE
Use JUnit 5 in documentation

### PR DESCRIPTION
**Pull request by minwoox**
- Use Junit5 for `advanced-unit-testing.mdx`
- Use `<type://class#method()>` where possible

### CODE REVIEW

File: site/src/pages/docs/advanced-unit-testing.mdx [hunk 1]
```diff
@@ -9,28 +9,28 @@ A unit test of a client or a service will require you to prepare two objects:
...
 
-public class MyJUnit4Test {
+class MyJUnit5Test {
 
     private MyClient client;
     private MyService service;
 
-    @Before
+    @BeforeEach
     public void setUp() {
```
Comment (ikhoon) [inline site/src/pages/docs/advanced-unit-testing.mdx:27]: nit:
```suggestion
    void setUp() {
```

File: site/src/pages/docs/advanced-unit-testing.mdx [hunk 2]
```diff
@@ -9,28 +9,28 @@ A unit test of a client or a service will require you to prepare two objects:
...
         client = ...;
         service = ...;
     }
 
     @Test
-    public void testClient() throws Exception {
+    void testClient() throws Exception {
```
Comment (ikhoon) [inline site/src/pages/docs/advanced-unit-testing.mdx:33]: Could remove?
```suggestion
    void testClient() {
```

File: site/src/pages/docs/advanced-unit-testing.mdx [hunk 3]
```diff
@@ -44,7 +44,7 @@ public class MyJUnit4Test {
     }
 
     @Test
-    public void testService() throws Exception {
+    void testService() throws Exception {
```
Comment (ikhoon) [inline site/src/pages/docs/advanced-unit-testing.mdx:47]: Ditto

File: site/src/pages/docs/advanced-dropwizard-integration.mdx [hunk 4]
```diff
@@ -271,7 +271,7 @@ passed to the <type://ArmeriaBundle>.
 ## Server Access Logs
 
 Armeria Server `Access Logging <server-access-log>` is enabled by default when using the Armeria Server.
-The default <type://AccessLogWriter> is `AccessLogWriter.common()`, but this can be changed via the follow
+The default <type://AccessLogWriter> is <type://AccessLogWriter#common()>, but this can be changed via the follow
```
Comment (trustin) [inline site/src/pages/docs/advanced-dropwizard-integration.mdx:274]: following

File: site/src/pages/docs/advanced-unit-testing.mdx (hunk 1 above)
Comment (minwoox) [reply site/src/pages/docs/advanced-unit-testing.mdx]: Fixed. 😉

File: site/src/pages/docs/advanced-unit-testing.mdx (hunk 2 above)
Comment (minwoox) [reply site/src/pages/docs/advanced-unit-testing.mdx]: Changed `.get()` to `.join()` and removed it. Thanks!

File: site/src/pages/docs/advanced-dropwizard-integration.mdx (hunk 4 above)
Comment (minwoox) [reply site/src/pages/docs/advanced-dropwizard-integration.mdx]: Fixed. 😉

**Pull request merged**